addons/hr_holidays/views/hr_leave_type_views.xml

```diff
@@ -55,41 +55,6 @@
         </field>
     </record>
 
-    <record id="view_holiday_status_tree" model="ir.ui.view">
-        <field name="name">hr.leave.type.tree</field>
-        <field name="model">hr.leave.type</field>
-        <field name="arch" type="xml">
-            <tree string="Leave Type">
-                <field name="name"/>
-                <field name="max_leaves"/>
-                <field name="leaves_taken"/>
-                <field name="remaining_leaves"/>
-            </tree>
-        </field>
-    </record>
-
-    <record id="hr_holiday_status_view_kanban" model="ir.ui.view">
-        <field name="name">hr.leave.type.kanban</field>
-        <field name="model">hr.leave.type</field>
-        <field name="arch" type="xml">
-            <kanban class="o_kanban_mobile">
-                <templates>
-                    <t t-name="kanban-box">
-                        <div t-attf-class="oe_kanban_global_click">
-                            <div>
-                                <strong><field name="name"/></strong>
-                            </div>
-                            <div>
-                                <span>Max Leaves: <field name="max_leaves"/></span>
-                                <span class="pull-right">Leaves Taken: <field name="leaves_taken"/></span>
-                            </div>
-                        </div>
-                    </t>
-                </templates>
-            </kanban>
-        </field>
-    </record>
```

Comment:
I get that why you remove the kanban view: The only two fields that were displayed are the ones that are moved on another model.

That said, you remove the kanban view but do not change the action to remove the kanban from the possible views for that model/menu, and therefore the kanban is still available, and it render a default kanban which is a bit useless:
![screenshot-330252-master-leave-type-compute-leaves-106910 runbot14 odoo com-2018 05 23-12-04-03](https://user-images.githubusercontent.com/5822488/40418087-b28efb4e-5e81-11e8-8089-007e479145a2.png)

I would remove the kanban view from the action so it no longer give the possibility to display the leaves types in a kanban view.